### PR TITLE
OD-391 [Fix] Fix accessibility issues

### PR DESCRIPTION
**Pull request by AndrRyaz**
@sofiiakvasnevska 
## Issue
OD-391 https://weboo.atlassian.net/browse/OD-391
## Description
- Added fixes for each lfd component
## Screenshots/screencasts
http://somup.com/creOQMbyFx
## Backward compatibility
This change is fully backward compatible.
## Reviewers
@upplabs-alex-levchenko @ivandevupp @YaroslavOvdii

### CODE REVIEW

File: js/layout-javascript/agenda-code.js [hunk 1]
```diff
@@ -199,6 +199,9 @@ DynamicList.prototype.attachObservers = function() {
         return;
       }
 
+      $(this).parents('.new-agenda-search-filter-overlay').removeClass('display');
+      $('.section-top-wrapper, .agenda-cards-wrapper, .dynamic-list-add-item').removeClass('hidden');
```
Comment (tonytlwu) [inline js/layout-javascript/agenda-code.js:203]: Ues `_this.$container` to restrict selection within the widget instance.

File: js/layout-javascript/small-card-code.js [hunk 2]
```diff
@@ -333,6 +333,8 @@ DynamicList.prototype.attachObservers = function() {
         return;
       }
 
+      $(event.target).parents('.small-card-list-wrapper').addClass('hidden');
```
Comment (tonytlwu) [inline js/layout-javascript/small-card-code.js:336]: `$(event.target)` and `$(this)` are used multiple times in the same closure. Use a variable.

**Review by tonytlwu — CHANGES_REQUESTED** [review]

File: js/layout-javascript/agenda-code.js (hunk 1 above)
Comment (AndrRyaz) [reply js/layout-javascript/agenda-code.js]: @tonytlwu Fixed

File: js/layout-javascript/small-card-code.js (hunk 2 above)
Comment (AndrRyaz) [reply js/layout-javascript/small-card-code.js]: @tonytlwu Fixed

**Comment by AndrRyaz** [comment]
@tonytlwu all comments are fixed

File: js/layout-javascript/small-card-code.js [hunk 3]
```diff
@@ -325,18 +325,19 @@ DynamicList.prototype.attachObservers = function() {
       }
 
       var _that = $(this);
+      var $el = $(event.target);
```
Comment (tonytlwu) [inline js/layout-javascript/small-card-code.js:328]: `$(event.target)` and `$(this)` are the same, which means `_that` and `$el` are the same. Pick one, and whichever you go with, make sure the variable is prefixed with `$`.

**Review by tonytlwu — CHANGES_REQUESTED** [review]

File: js/layout-javascript/small-card-code.js (hunk 3 above)
Comment (AndrRyaz) [reply js/layout-javascript/small-card-code.js:327]: @tonytlwu fixed

File: js/layout-javascript/agenda-code.js [hunk 4]
```diff
@@ -606,6 +609,8 @@ DynamicList.prototype.attachObservers = function() {
       var result;
       var id = _this.$container.find('.agenda-detail-wrapper[data-entry-id]').data('entry-id');
 
+      $('.agenda-list-holder').removeClass('hidden');
```
Comment (tonytlwu) [inline js/layout-javascript/agenda-code.js:612]:
```suggestion
      _this.$container.find('.agenda-list-holder').removeClass('hidden');
```

Comment (AndrRyaz) [reply js/layout-javascript/agenda-code.js]: @tonytlwu i researched all my pr and find some lines where this code is needed and added it

**Review by tonytlwu — APPROVED** [review]
@sofiiakvasnevska This is ready for testing

Risk: Medium
Affected: All clickable LFD features
Verify: All clickable LFD features are working as expected

Squash and merge: Use PR title

**Pull request merged**